packages/documentation-framework/components/example/example.css

```
@@ -101,32 +67,32 @@
 }
 
 .pf-v6-c-badge.ws-beta-badge {
-/*   --pf-v6-c-badge--m-unread--BackgroundColor: var(--pf-v6-global--BackgroundColor--100); */
-/*   --pf-v6-c-badge--m-unread--Color: var(--pf-v6-global--primary-color--100); */
-/*   border: var(--pf-v6-global--BorderWidth--sm) solid var(--pf-v6-global--primary-color--100); */
+   --pf-v6-c-badge--m-unread--BackgroundColor: var(--pf-t--global--background--color--floating--default);
+   --pf-v6-c-badge--m-unread--Color: var(--pf-t--global--color--status--unread--default--default);
+   border: var(--pf-t--global--border--width--regular) solid var(--pf-t--global--border--color--nonstatus--blue--default);
 }
 
 .ws-prop-required {
-/*   color: var(--pf-v6-global--danger-color--100); */
+  color: var(--pf-t--global--text--color--status--danger--default);
 }
 
 .ws-full-page-utils {
   position: fixed;
   right: 0;
   bottom: 0;
   padding: var(--pf-t--global--spacer--lg);
-/*   z-index: var(--pf-v6-global--ZIndex--2xl); */
+  z-index: var(--pf-t--global--z-index--600);
```

whenever possible it's better to use the most semantic version of a token. The ones that end in numbers like `--[100, 200, etc]` are base tokens and we're not protecting those from breaking changes, not like it's a big deal for org though. And for z-index it's a 1:1 match with the previous tokens, just with the different prefix (`pf-t--global` instead of `pf-[v]-global`). 

```suggestion
  z-index: var(--pf-t--global--z-index--2xl);
```